src/main/java/com/personal/financeManager/documentProcessor/services/impl/FeatureTemplateServiceImpl.java

```diff
@@ -31,7 +31,6 @@ public SuccessResponse createTemplate(FeatureTemplateBean featureTemplateBean) {
         JSONObject data = new JSONObject(featureTemplateBean.getData());
         
         // #3) Generate DocumentID
```

Comment:
You didn't want to modify this file, as you have removed a line, it's not going to be acceptable further.
Please use the **"git status"** command before adding or committing and see how many files or code you have modified.